playbooks/incident-handling.md

```diff
@@ -0,0 +1,458 @@
+
+<details>
+  <summary>Examples of worthy incident reports:</summary>
+
+- a business-critical feature (such as authentication, search, inquiry, bidding, or payment) is disrupted
+- simultaneous support requests from users or partners indicating a pattern
+- time-sensitive disruptions, such as preventing an auction from opening or closing
+  </details>
+
+### 2. Raise
+
+1. **Track the incident** by creating an incident in [OpsGenie](https://artsy.app.opsgenie.com/incident/list) based
+   on the available information
+
+- Select _Artsy_ as the impacted service.
+- Enter a short description and link to the #incidents thread in the summary field.
+- Use your judgment to select a suitable _Priority_ level, based on the incident's severity.
+- Take your best guess at the other information; it can always be edited later.
+
+2. **Create a Slack channel** dedicated to the incident's technical resolution
+
+- Follow the **Create Slack channel** link from the OpsGenie incident page and create a `#inc-*` channel named with
+  the incident's number.
+- Invite the other on-call engineer and any other active responders to the new channel.
+- In the original #incidents thread, share a link to the new #inc-\* channel and _direct any further technical
+  discussion there_.
+
+4. If the incident affects a critical public function (e.g. search) or a broad category of users (e.g., all
+   partners), **update the [Artsy status page](https://status.artsy.net)** via the
+   [StatusPage.io](https://manage.statuspage.io/pages/hmhlbjlmdhgh/incidents) account (details in 1Password). The
+   update should _briefly_ acknowledge the issue, focusing on public impact
+   ([examples](https://twitter.com/stripestatus)). Technical details or system code names are not relevant or
+   appropriate.
+
+### 3. Address
+
+#### Roles
+
+The first on-call engineer on the scene should
+[take the role of **Incident Commander**](<(https://docs.opsgenie.com/docs/incident-response-roles)>) unless
+there's another volunteer or naturally responsible candidate. Your responsibility in this role is to _facilitate_ a
+successful resolution, not to solve the problem yourself.
+
+If both on-call engineers are available, the 2nd should take the role of **Communications Officer**, handling
+updates to internal stakeholders (on the #incidents thread) and external audiences (on the status page), as well as
+recording relevant information (insights, links...) on the incident timeline. If both engineers aren't available,
+the incident commander can perform both roles or ask for another volunteer.
+
+#### Diagnose
+
+Discuss the incident's resolution in the dedicated Slack channel. Tips:
+
+- Scan [recent releases](https://github.com/pulls?q=is%3Amerged+is%3Apr+archived%3Afalse+org%3Aartsy+deploy) that
+  may be responsible.
+  [Roll back](https://github.com/artsy/hokusai/blob/master/docs/Command_Reference.md#how-to-do-a-rollback) the
```

Comment:
We could add a line here to help on-call engineers decide whether to roll back or revert the change as a first action, since this has been a sticking point in the past.